modules/ROOT/pages/log-search-query-syntax.adoc

```diff
@@ -76,6 +76,7 @@ The `NOT` operator can't be used with just one term. For example, the following
 `"anypoint monitoring" -"results"`
 |===
 
+
 == Special Characters
```

Comment:
Again, is this a peer of the other headings? Does it make sense for them to be peers under their container topic? Does the container topic short description prepare the audience for each of these subtopics?